pip installation error

while trying to run the command
`pip install git+https://github.com/Maluuba/nlg-eval.git@master`
I get error as:
```
ERROR: Command errored out with exit status 1:
     command: /home/mshaikh2/anaconda3/envs/tf-gpu/bin/python3.5 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-4t0hee9u/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-4t0hee9u/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-f77lviyt
         cwd: /tmp/pip-req-build-4t0hee9u/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-4t0hee9u/setup.py", line 24, in <module>
        reqs = [str(ir.req) for ir in install_reqs]
      File "/tmp/pip-req-build-4t0hee9u/setup.py", line 24, in <listcomp>
        reqs = [str(ir.req) for ir in install_reqs]
    AttributeError: 'ParsedRequirement' object has no attribute 'req'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Please help me fix it, as I need to use all the evaluations.

Thanks for your help again, following the instructions and rerunning the setup fixed the files.
However, I now have an assertion error!

I am using the script for multiple examples:
```
from nlgeval import NLGEval
nlgeval = NLGEval()  # loads the models
metrics_dict = nlgeval.compute_metrics(references, hypothesis)
```
Below are two reference sentences corresponding to two hypothesis sentences respectively, i.e. ref[0] corresponds to hyp[0] and ref[1] corresponds to hyp[1]:

```
references = ['heart size within normal limits. no typical findings of pulmonary edema. no pneumothorax.',
  'cardiac size within normal limits. no focal consolidation or pneumothorax. the visualized osseous structures appear intact.']
```
 ```
hypothesis = ['both lungs are clear bilaterally . specifically , no evidence of focal consolidation , pneumothorax , or pleural effusion . cardiomediastinal silhouette is unremarkable . visualized osseous structures appear intact . there are no acute bony abnormality .',
  'both lungs are clear bilaterally . specifically , no evidence of focal consolidation , pneumothorax , or pleural effusion . cardiomediastinal silhouette is unremarkable . visualized osseous structures appear intact . no acute bony abnormality .']
```

I get an assertion error:
```
AssertionError                            Traceback (most recent call last)
<ipython-input-129-06535c64bccb> in <module>()
----> 1 metrics_dict = nlgeval.compute_metrics(references, hypothesis)

~/anaconda3/envs/tf-gpu/lib/python3.5/site-packages/nlgeval/__init__.py in compute_metrics(self, ref_list, hyp_list)
    290         refs = {idx: strippedlines for (idx, strippedlines) in enumerate(ref_list)}
    291         hyps = {idx: [lines.strip()] for (idx, lines) in enumerate(hyp_list)}
--> 292         assert len(refs) == len(hyps)
    293 
    294         ret_scores = {}

AssertionError: 
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,6 @@ sudo: required
 language: python
 python:
   - "3.6"
-  - "2.7"
 branches:
   only:
   - master

--- a/requirements_py2.txt
+++ b/requirements_py2.txt
@@ -1,5 +1,5 @@
 click>=6.3
-nltk>=3.1
+nltk==3.1
 numpy>=1.11.0<=1.17
 psutil>=5.6.2
 requests>=2.19

--- a/setup.py
+++ b/setup.py
@@ -21,7 +21,10 @@ if __name__ == '__main__':
     if sys.version_info[0] < 3:
         requirements_path = 'requirements_py2.txt'
     install_reqs = parse_requirements(requirements_path, session=False)
-    reqs = [str(ir.req) for ir in install_reqs]
+    try:
+        reqs = [str(ir.req) for ir in install_reqs]
+    except:
+        reqs = [str(ir.requirement) for ir in install_reqs]
 
     setup(name='nlg-eval',
           version='2.3',

